src/test/java/seedu/address/storage/JsonAdaptedSkillTest.java

```diff
@@ -1,4 +1,104 @@
 package seedu.address.storage;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import static seedu.address.storage.JsonAdaptedSkill.MISSING_FIELD_MESSAGE_FORMAT;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalSkill.REACT;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.item.field.Level;
+import seedu.address.model.item.field.Name;
+import seedu.address.model.tag.Tag;
+import seedu.address.testutil.TypicalSkill;
+
 public class JsonAdaptedSkillTest {
+
+    private static final String VALID_NAME = REACT.getName().toString();
+    private static final String VALID_LEVEL = REACT.getLevel().toString();
+    private static final String VALID_ID = String.valueOf(REACT.getId());
+
+    private static final List<JsonAdaptedTag> EMPTY_TAGS = new ArrayList<>();
+    private static final List<JsonAdaptedTag> VALID_TAGS = REACT.getTags().stream()
+            .map(JsonAdaptedTag::new)
+            .collect(Collectors.toList());
+
+    private static final String INVALID_NAME = "Re@ct";
+    private static final String INVALID_LEVEL = "Professional";
+    private static final String INVALID_ID = "2a";
+    private static final String INVALID_TAG = "?cool";
```

Comment:
To be consistent with the other tests, we should have used the public constants in `CommandTestUtil`, but I'm fine keeping it like this in view of time constraints.

Reply:
`CommandTestUtil` is under the `logic.command` subpackage so I think we shouldn't let test from a different component access it.